lib/templates/accountTemplate.js

```diff
@@ -86,23 +95,59 @@ class AccountTemplate {
     fsUtils.writeConfig(this.TEMPLATE_TYPE, name, templateConfig);
   }
 
-  static #prepareConfigDetails(template, existingConfig = {}) {
+  static #prepareConfigDetails(type, envId, template, existingConfig = {}) {
     const attributes = this.CONFIG_ITEMS.reduce((acc, attribute) => {
       if (template.hasOwnProperty(attribute)) {
         acc[attribute] = template[attribute];
       } else if (existingConfig?.hasOwnProperty(attribute)) {
         acc[attribute] = existingConfig[attribute];
       }
+
       return acc;
     }, {});
+
+    // We need to also create the mapping_list_ranges, which could be different per firm & partner
+    const existingMappingRanges = existingConfig.mapping_list_ranges || [];
+    const updatedMappingRanges = [...existingMappingRanges];
+
+    for (let mappingListRange of template.mapping_list_ranges) {
+      // Find the existing mapping list range, if any
+      const existingIndex = existingMappingRanges.findIndex(
+        (existingMappingListRange) =>
+          existingMappingListRange.account_mapping_list_id ===
```

Comment:
At the current state this only supports firm-level ranges. Partner-level ones have keys that start with `partner_` (as in `partner_account_mapping_list_id`)

Reply:
It should support both the mapping lists for firms & partners; in the config this will be stored in an example like below where there's combination of both:
![image](https://github.com/silverfin/silverfin-cli/assets/99254741/b39892bc-c0b5-4d79-93fc-c94cc062fc15)